src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -361,9 +369,9 @@ export class CreateScenariosComponent implements OnInit {
   }
 
   disableForms() {
-    this.formGroups.forEach((form) => {
-      form.disable();
-    });
+    this.formGroups[1].disable();
```

Comment:
It may be worth also disabling the name formgroup for now since there  isn't  a way for users to save scenario name changes atm

Reply:
ah true I leaved this half baked, thanks for the heads up!

Reply:
thanks for catching this @sachiyuan - seems like looping on the forms was causing the weird issue- there is a third form that Im not sure we are using? but If I disable that one the selection on the radio buttons gets lost.

The good news is that I dont need the hacky extra `disable` property

Reply:
Is the third form the project areas one? That one is was kept from the legacy code bc it will likely be used when y'all add the ability for users to upload their own project areas to the scenario for analysis. Odd that it's wiping out the radio button selection though

Yay for not needing hacky bits!